sphinxcontrib/confluencebuilder/storage/translator.py

```diff
@@ -125,6 +124,12 @@ def __init__(self, document, builder):
         self.colspecs = []
         self._tocdepth = self.state.toctree_depth(self.docname)
 
+        if isinstance(config.confluence_default_table_width, string):
+            table_width, twu = extract_length(config.confluence_default_table_width)
+            self._v2_default_table_width = convert_length(table_width, twu)
```

Comment:
If the resulting `data-table-width` does not take a percentage value, `pct=False` should be set in this call. If it does, please disregard.

Reply:
I added a check in checks.py so that percentage is not allowed.